sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/SclService.java

```diff
@@ -392,44 +406,42 @@ public static SclRootAdapter importSTDElementsInSCD(@NonNull SclRootAdapter scdR
 
 
         // List all Private and remove duplicated one with same iedName
-        Map<String, TPrivate> tPrivateMap = createMapIEDNameAndPrivate(scdRootAdapter);
+        Map<String, TPrivate> mapIEDNameAndPrivate = createMapIEDNameAndPrivate(scdRootAdapter);
 
-        Map<String, Pair<TPrivate, List<SCL>>> stringListMap = createMapICDSystemVersionUuidAndSTDFile(stds);
-        for (Map.Entry<String, Pair<TPrivate, List<SCL>>> entry : stringListMap.entrySet()) {
-            if (entry.getValue().getRight().size() != 1){
+        Map<String, Pair<TPrivate, List<SCL>>> mapICDSystemVersionUuidAndSTDFile = createMapICDSystemVersionUuidAndSTDFile(stds);
+        for (Map.Entry<String, Pair<TPrivate, List<SCL>>> entry : mapICDSystemVersionUuidAndSTDFile.entrySet()) {
+            if (entry.getValue().getRight().size() != 1) {
```

Comment:
I suggest creating a DTO or an inner class to hold data instead of using `Pair`.
With `Pair`, it is difficult to remember what data is in on left and right. Here, it is difficult to know what `entry.getValue().getRight()` is.

At least, I suggest putting `entry.getValue().getRight()` in a variable with an explicit name.